docs/source/examples.rst

```diff
@@ -633,93 +670,3 @@ Advanced Projects
             self.play(Transform(grid_title, grid_transform_title))
             self.wait()
 
-.. manim:: SineCurveUnitCircle
```

Comment:
I removed SineCurveUnitCircle, as it is a bit too long as an example on the main example gallery and not programmed in the most efficient way.

Reply:
If you say that it wasn't programmed in the most efficient way, the example should be improved -- but I disagree with completely removing it. I think it makes sense for our example gallery to include a few longer / more complex examples as well.

Reply:
e.g. the part :
```py
def add_x_labels(self):
            x_labels = [
                MathTex("\pi"), MathTex("2 \pi"),
                MathTex("3 \pi"), MathTex("4 \pi"),
            ]

            for i in range(len(x_labels)):
                x_labels[i].next_to(np.array([-1 + 2*i, 0, 0]), DOWN)
                self.add(x_labels[i])
```
I don't like too much, and I currently don't have the time to rewrite it by myself.
Also, I think it does not add too much value to the manim feature overview.
Instead, we could add some other classes, e.g. the Graph class or an example to show how to use rate functions.

Reply:
> it does not add too much value to the manim feature overview

I disagree. It shows how you can combine several concepts to construct a full-fledged animation.

I don't have time to rewrite it either, and there is still a lot of work to do for the release. I'm happy to keep discussing this, but I feel that this change is too controversial to be merged quickly.

Reply:
sounds good.
And I also I agree that we should show a full-fledged animation, but maybe not exactly this one.
But this can be discussed after the release.